src/utils.js

```diff
@@ -38,23 +38,23 @@ function buildWorkerCode(source, sourcemap = null, inline = true, preserveSource
             return `\
 /* eslint-disable */\n\
 import {createInlineWorkerFactory} from 'rollup-plugin-web-worker-loader::helper';\n\
-const WorkerFactory = createInlineWorkerFactory(${source.substring(0, source.length - 1)}, ${sourcemap ? `'${sourcemap.toUrl()}'` : 'null'});\n\
+var WorkerFactory = createInlineWorkerFactory(${source.substring(0, source.length - 1)}, ${sourcemap ? "'" + sourcemap.toUrl() + "'" : 'null'});\n\
```

Comment:
Here the quotes rather than template literals is not needed as that code will not make it into the final compiled file

Reply:
Gotcha - is that because it's anticipated that we wouldn't normally use sourcemaps in production?

Reply:
the code inside the escape character (`${}`) runs in node during compilation so the result of ``${sourcemap ? `'${sourcemap.toUrl()}'` : 'null'}`` would be resolved at compile time

Reply:
Understood! Thanks :)